tests/avocado/parameters.yaml

```diff
@@ -0,0 +1,59 @@
+architecture: !mux
+    x86_64:
```

Comment:
Anyway to this file, I'd probably recommend something like `variants.yaml` to emphasize this creates variants and I'd not comment-out the variants. I expect people by default running it without variants and when they use `-m` they should use `--mux-filter-out` to omit unnecessary variants.

I don't see this as a good source for default params, that should be in tests. I see this file as a source of possible (default) variants.